examples/3d_drawing/plot_3d_animation_basic.py

```diff
@@ -0,0 +1,60 @@
+"""
+===============
+Basic Animation
+===============
+
+NetworkX supports several 3D layout functions for visualization.
+These layouts can be combined with :external+matplotlib:doc:`api/animation_api`
+for 3D graph visualization.
+
+This example shows a basic animation incrementing the camera view.
```

Comment:
Add from my original PR: 
```
Following 
https://sphinx-gallery.github.io/stable/auto_examples/plot_8_animations.html
using frame rotation of an initial plot of a graph as in 
https://matplotlib.org/stable/api/animation_api.html
```

Reply:
I removed this intentionally, on the idea that the networkx gallery should really focus on network visualization specifically. Matplotlib's and sphinx-gallery's examples and reference docs are a much better source for this info.